### DISCUSSION TITLE
Postgres Portal 500 Error

### DISCUSSION THREAD

**Discussion by norbusonam** [vercel/community#4454] [Account Issue] [answered]
### Question

I did something dumb. I was trying to drop my vercel postgres db, so it would have a fresh start. I went onto my db server with `psql` and tried to drop it but it had two connections. I tried to kill those connections and only one successfully closed. But now, whenever I go that db on vercel.com, I get a 500 error. Any ideas on how to fix this? If someone from vercel could just delete the db that would work for me too.

**Comment by norbusonam** [comment]
Here's the error details in case it helps

<img width="613" alt="Screenshot 2023-10-12 at 12 38 31 PM" src="https://github.com/vercel/community/assets/43245807/fc70b0c7-9a24-4dd9-8029-882b9c5297ab">

**Reply by amyegan** [reply]
Thanks for sharing that! I asked the Vercel Postgres team for help with this and I'll post updates here when I know more.

**Reply by norbusonam** [reply]
store_weK4TkjBJM6kmp0Z

this should be the db id if someone can just delete it so I can make a new one :)

**Reply by amyegan** [reply, marked as answer]
Done! You should be all clear to create a new one now 😄

**Reply by norbusonam** [reply]
Awesome, thank you so much!
